app/src/main/java/com/example/internetshop/data/repository/CategoryRepositoryImpl.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.internetshop.data.repository
+
+import com.example.internetshop.data.response.CategoryResponse
+import com.example.internetshop.data.response.mapper.CategoryResponseMapper
+import com.example.internetshop.data.retrofitapi.CategoryApi
+import com.example.internetshop.domain.data.model.Category
+import com.example.internetshop.domain.data.repository.CategoryRepository
+import io.reactivex.Single
+import javax.inject.Inject
+
+class CategoryRepositoryImpl @Inject constructor(
+    private val categoryApi: CategoryApi,
+    private val categoryResponseMapper: CategoryResponseMapper
+) :
+    CategoryRepository {
```

Comment:
Looks like formatting off here. Please format the file

Reply:
Fixed